hawkular-metrics/Dockerfile

```diff
@@ -77,10 +77,10 @@ COPY logging.properties $JBOSS_HOME/standalone/configuration/logging.properties
 
 # Install Hawkular-Metrics Python client, only available on EPEL
 USER root
-RUN yum -y install epel-release \
-    && yum -y install python-pip \
-    && yum clean all \
-    && pip install --no-cache-dir 'hawkular-client==0.4.4'
+#RUN yum -y install epel-release \
+#    && yum -y install python-pip \
+#    && yum clean all \
+#    && pip install --no-cache-dir 'hawkular-client==0.4.4'
```

Comment:
@mwringe  +1, I'll revert that.  It's like that because I was having trouble building one day because I couldn't pull down epel-release from the repo...